src/Autofac/Autofac.csproj

```diff
@@ -4,7 +4,7 @@
     <Description>Autofac is an IoC container for Microsoft .NET. It manages the dependencies between classes so that applications stay easy to change as they grow in size and complexity.</Description>
     <!-- VersionPrefix patched by AppVeyor -->
     <VersionPrefix>0.0.1</VersionPrefix>
-    <TargetFrameworks>netstandard2.1;netstandard2.0;net5.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;netstandard2.0;net5.0;net6.0</TargetFrameworks>
     <LangVersion>latest</LangVersion>
     <Nullable>enable</Nullable>
     <NoWarn>$(NoWarn);CS1591;IDE0008</NoWarn>
```

Comment:
Down on line 31 you'll see `<Features>` - I can't make a comment that far down in the PR interface. The way analyzers work in .NET 6 is different than in other frameworks. We have to turn some things off and turn some things on.

- `<Features>` needs to be removed. That's obsolete now.
- `<AnalysisMode>AllEnabledByDefault</AnalysisMode>` needs to be added where the `<Features>` line is.

This also needs to be done in the test projects that have analyzers. I don't know if they all have `<Features>` but that `<AnalysisMode>` line needs to be there.

This may cause some new warnings to show up. All warnings need to be resolved as part of the .NET upgrade - we don't allow builds with warnings. This may be more than you want to take on.

Reply:
Done